build.info

```diff
@@ -1,6 +1,10 @@
 # Note that some of these directories are filtered in Configure.  Look for
 # %skipdir there for further explanations.
-SUBDIRS=crypto ssl apps test util tools fuzz engines providers doc
+$DEPRECATED_SUBDIRS=
+IF[{- !$disabled{'deprecated-3.0'} -}]
+  $DEPRECATED_SUBDIRS=engines
+ENDIF
+SUBDIRS=crypto ssl apps test util tools fuzz providers doc $DEPRECATED_SUBDIRS
```

Comment:
You forget that these things are cumulative, so you could simply to it like this:

```
SUBDIRS=crypto ssl apps test util tools fuzz providers doc
IF[{- !$disabled{'deprecated-3.0'} -}]
  SUBDIRS=engines
ENDIF
```

Reply:
Okay, done.